packages/api/src/command/medical/admin/upload-doc.ts

```diff
@@ -1,15 +1,17 @@
 import { DocumentReference } from "@medplum/fhirtypes";
 import dayjs from "dayjs";
 import { makeFhirApi } from "../../../external/fhir/api/api-factory";
-import { createDocReferenceContent } from "../../../external/fhir/document";
+import { createDocReferenceContent, getFHIRDocRef } from "../../../external/fhir/document";
 import { metriportDataSourceExtension } from "../../../external/fhir/shared/extensions/metriport";
 import { Config } from "../../../shared/config";
-import { getOrganizationOrFail } from "../organization/get-organization";
+import { randomInt } from "../../../shared/numbers";
 import { getPatientOrFail } from "../patient/get-patient";
 
 const apiUrl = Config.getApiUrl();
 const docContributionUrl = `${apiUrl}/doc-contribution/commonwell/`;
 
+const smallId = () => String(randomInt(3)).padStart(3, "0");
```

Comment:
nit: `3` could go in a named const

Reply:
yeah, but its just an internal endpoint and this is only used here, so for the sake of finishing this before getting AFK I'll leave like this :)